### PR TITLE
add search button and link it to function

### PR DESCRIPTION
**Pull request by alexresiga**
#4

### CODE REVIEW

**Comment by NitinNair89** [comment]
Thanks! Can you setup a GitHub page pointing to your fork repo's master branch so that I can view the UI with your code changes?

**Comment by alexresiga** [comment]
https://alexresiga.github.io/TheHungryRecipes/index.html

File: index.html [hunk 1]
```diff
@@ -47,11 +47,12 @@ <h4>Confused what to cook? <br/>
                           <small>Search a recipe or try a random recipe!</small>
                         </h4>
                         <div class="row">
-                          <div class="eight columns">
+                          <div class="four columns">
```
Comment (NitinNair89) [inline index.html]: Let's not change the grid column size

File: index.html [hunk 2]
```diff
@@ -47,11 +47,12 @@ <h4>Confused what to cook? <br/>
...
                             <input class="u-full-width" type="text" placeholder="Search recipes" id="searchRecipe" />
                           </div>
-                          <div class="four columns" style="position:relative;" id="randomMealBtnContainer">
+                          <div class="eight columns" style="position:relative;" id="randomMealBtnContainer">
```
Comment (NitinNair89) [inline index.html]: Let's not change the grid column size

**Review by NitinNair89 — CHANGES_REQUESTED** [review]
> https://alexresiga.github.io/TheHungryRecipes/index.html

The interface is broken. Please make the below changes:
- "Search Recipe" should be the button text and position it below the "Random Recipe" button
- Add the chef hat icon to the search recipe button
- Do not change the grid column size, keep it as-is

FireShot Capture 004 - The Hungry Recipes - alexresiga github io

File: index.html (hunk 1 above)
Comment (alexresiga) [reply index.html]: solved see new commit

**Review by NitinNair89 — CHANGES_REQUESTED** [review]
I checked the live site on the link you previously shared. The alignment seems a little off. Let's do it this way:
- Create a row inside the eight columns div similar to the structure made for the input field
- Keep the chef hat icon for the random recipe button

**Comment by alexresiga** [comment]
> I checked the live site on the link you previously shared. The alignment seems a little off. Let's do it this way:

the alignment is a bit off because the buttons don't have a fixed width they expand with the text inside them

> * Create a row inside the eight columns div similar to the structure made for the input field

i don't understand what you mean by this
> * Keep the chef hat icon for the random recipe button

ok

**Comment by NitinNair89** [comment]
> i don't understand what you mean by this

Currently, the h4 is floating and occupies full width of its container. We can create a row class div with the same structure as for input field

**Current code:**
`<h4>Confused what to cook? <br/><small>Search a recipe or try a random recipe!</small></h4>`

**Suggestion:**
```
<div class="row">
  <div class="eight columns">
    <h4>Confused what to cook? <br/><small>Search a recipe or try a random recipe!</small></h4>
  </div>
  <div class="four columns">
      Random Recipe button
  </div>
</div>
```
Can you give this a try and see if it solves alignment issues? :slightly_smiling_face:

**Comment by NitinNair89** [comment]
Have you tested everything?

**Comment by alexresiga** [comment]
yes, you can see oh the gh pages link

**Comment by NitinNair89** [comment]
> yes, you can see oh the gh pages link

Looks fine. There's a bug, I will open a separate issue and assign it to you. Is that fine?

**Comment by NitinNair89** [comment]
Closes #4

**Comment by alexresiga** [comment]
> Looks fine. There's a bug, I will open a separate issue and assign it to you. Is that fine?

sure

**Pull request merged**